akka-stream/src/main/scala/akka/stream/Fusing.scala

```diff
@@ -34,6 +34,12 @@ object Fusing {
   def aggressive[S <: Shape, M](g: Graph[S, M]): FusedGraph[S, M] = Impl.aggressive(g)
 
   /**
+   * Return the StructuralInfo for this Graph without any fusing
+   */
+  def structuralInfo[S <: Shape, M](g: Graph[S, M], attributes: Attributes): StructuralInfoModule =
```

Comment:
this should probably have an `INTERNAL API` marker in the docs, since `StructuralInfoModule` is in the `impl` package